acasclient/acasclient.py

```diff
@@ -1066,9 +1066,7 @@ def register_sdf(self, file, userName, mappings, prefix=None, dry_run=False):
             response = self.register_sdf_request(request)
         report_files = []
         for file in response[0]['reportFiles']:
-            filePath = "/dataFiles/cmpdreg_bulkload/{}".format(
-                PurePath(Path(file)).name)
-            report_files.append(self.get_file(filePath))
+            report_files.append(self.get_file(f"/dataFiles/{file}"))
```

Comment:
Bulkloader now returns the file path instead of just the file name

Reply:
We should document this for the DataTeam in release notes. I am pretty sure they're parsing responses from SDF loads